src/constants/main.constants.ts

```diff
@@ -47,3 +47,6 @@ export const BANNER = `-----                    __   .__.__  .__
 --        |___|  /   __/|__|_ \\__|____/____/
 -------        \\/|__|        \\/
 `;
+
+export const STREAM_ENCODING = 'utf8';
+export const WINDOWS = 'win32';
```

Comment:
Ambiguous Name. Maybe "WINDOWS_PLATFORM" will better?

Reply:
Change as you wish